src/app/pages/property-details/confirmation-modal/confirmation-modal.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+import { ActivatedRoute, Router } from '@angular/router';
+import { PropertyDetailService } from 'src/app/services/property-detail/property-detail.service';
+
+@Component({
+  selector: 'app-confirmation-modal',
+  templateUrl: './confirmation-modal.component.html',
+  styleUrls: ['./confirmation-modal.component.scss']
+})
+export class ConfirmationModalComponent implements OnInit {
+  isModalClosed = true;
```

Comment:
@MBenedicte since typescript is strongly typed, I suggest you consider stating the type for `isModalClosed` to boolean